atomicfu-transformer/src/main/kotlin/kotlinx/atomicfu/transformer/AtomicFUTransformer.kt

```diff
@@ -240,8 +262,8 @@ class AtomicFUTransformer(
         return result
     }
 
-    private inner class CollectorCV : CV(null) {
-        override fun visitField(access: Int, name: String, desc: String, signature: String?, value: Any?): FieldVisitor? {
+    private inner class CollectorCV: CV(null) {
```

Comment:
There should be a space before colon (`:`) when it is used to specify a list of super classes and super-interfaces (see Kotlin code style).